packages/next-preact/alias.js

```diff
@@ -1,6 +1,6 @@
 const moduleAlias = require('module-alias')
 
 module.exports = () => {
-  moduleAlias.addAlias('react', 'preact-compat')
+  moduleAlias.addAlias('react', __dirname + '/preact-compat')
```

Comment:
Can we add `.js` to the end of these? It wasn't immediately obvious to me that we had our own file named `preact-compat.js` instead of the module itself.

Reply:
I am sorry, it was a bit incomprehensible.
added `.js` to the end of these.